library/ix-dev/community/homebridge/README.md

```diff
@@ -0,0 +1,18 @@
+# Homebridge
+
+- This container runs as `root` user.
+
+If host networking is enabled, you must disable mDNS on the host.
```

Comment:
@anodos325 would this be an issue?

Reply:
Time Machine functionality is lost if mDNS is turned off on host as well as the convenience MacOS users have by truenas magically showing up in their network bar

Reply:
Is this acceptable?

Reply:
@stavros-k let's not allow auto device discovery then and re-consider when we get requests etc?

Reply:
Updated, note that it still relies on #1124 for the external Interfaces